workflow-templates/java-multimodule-project-build-deploy-pipeline.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Don't forge to add a link here to the workflow README.md from bw-workflow-actions once you've completed it